ui/src/features/scans/scansSchemas.ts

```diff
@@ -381,7 +381,6 @@ const scanResultsSchema: Yup.ObjectSchema<ScanResults> = Yup.object()
 	.defined();
 
 export const scanIdSchema = Yup.string()
-	.defined()
```

Comment:
schema used for validating scan ids & batch ids. removed `defined()` in this common schema so batch ids can be missing in the scan data. where scan id fields are validated, they already include their own `defined()` check